Source/qol/monhealthbar.cpp

```diff
@@ -63,46 +63,45 @@ void DrawMonsterHealthBar(const CelOutputBuffer &out)
 
 	const int width = healthBox.w();
 	const int height = healthBox.h();
-	int xPos = (gnScreenWidth - width) / 2;
+	Point position = { (gnScreenWidth - width) / 2, 18 };
 
 	if (CanPanelsCoverView()) {
 		if (invflag || sbookflag)
-			xPos -= SPANEL_WIDTH / 2;
+			position.x -= SPANEL_WIDTH / 2;
 		if (chrflag || questlog)
-			xPos += SPANEL_WIDTH / 2;
+			position.x += SPANEL_WIDTH / 2;
 	}
 
-	const int yPos = 18;
 	const int border = 3;
 
 	const int maxLife = std::max(mon._mmaxhp, mon._mhitpoints);
 
-	DrawArt(out, xPos, yPos, &healthBox);
-	DrawHalfTransparentRectTo(out, xPos + border, yPos + border, width - (border * 2), height - (border * 2));
+	DrawArt(out, position, &healthBox);
+	DrawHalfTransparentRectTo(out, position.x + border, position.y + border, width - (border * 2), height - (border * 2));
 	int barProgress = (width * mon._mhitpoints) / maxLife;
 	if (barProgress != 0) {
-		DrawArt(out, xPos + border + 1, yPos + border + 1, &health, 0, barProgress, height - (border * 2) - 2);
+		DrawArt(out, position + Point { border + 1, border + 1 }, &health, 0, barProgress, height - (border * 2) - 2);
 	}
 
 	if (sgOptions.Gameplay.bShowMonsterType) {
 		Uint8 borderColors[] = { 248 /*undead*/, 232 /*demon*/, 150 /*beast*/ };
 		Uint8 borderColor = borderColors[mon.MData->mMonstClass];
 		int borderWidth = width - (border * 2);
-		UnsafeDrawHorizontalLine(out, { xPos + border, yPos + border }, borderWidth, borderColor);
-		UnsafeDrawHorizontalLine(out, { xPos + border, yPos + height - border - 1 }, borderWidth, borderColor);
+		UnsafeDrawHorizontalLine(out, { position.x + border, position.y + border }, borderWidth, borderColor);

```

Comment:
I thought about doing this but then there are other computations with border all around this line that would look different. I preferred not to touch it right now since we have to change `border` itself to be something else (perhaps a `Rectangle`) and do a massive refactor in these methods.